WordPress/Classes/ViewRelated/NUX/NUXSubmitButton.swift

```diff
@@ -0,0 +1,120 @@
+import UIKit
+import WordPressShared
+
+let NUXSubmitButtonDisabledAlpha = CGFloat(0.25)
+
+/// A stylized button used by NUX controllers. The button presents white text 
+/// surrounded by a white border.  It also can display a `UIActivityIndicatorView`.
+///
+@objc class NUXSubmitButton : UIButton
```

Comment:
I looked at this some.  They are functionally similar, but there doesn't look like a huge amount of overlap.  I'm not seeing where we'd gain much by extracting a common super class.